apps/server/src/modules/board/controller/mapper/file-element-response.mapper.ts

```diff
@@ -0,0 +1,31 @@
+import { FileElement } from '@shared/domain';
+import { ContentElementType } from '../../types/content-element-type.enum';
+import { FileElementContent, FileElementResponse, TimestampsResponse } from '../dto';
+import { BaseResponseMapper } from './base-mapper.interface';
+
+export class FileElementResponseMapper implements BaseResponseMapper {
+	private static instance: FileElementResponseMapper;
+
+	public static getInstance(): FileElementResponseMapper {
```

Comment:
I am unsure if we make technical correctly over instance. 
vs
remove export from class and add this line
export const fileElementResponseMapper = new FileElementResponseMapper();
@Metauriel